docs/source/operations/projections/s2.rst

```diff
@@ -0,0 +1,109 @@
+.. _s2:
+
+********************************************************************************
+S2
+********************************************************************************
+
++---------------------+----------------------------------------------------------+
+| **Classification**  | Azimuthal                                                |
```

Comment:
Is it realy azimuthal ? If not Miscellaneous would be more appropriate

Reply:
In the cases when "none" or "linear" is chosen for the area-normalization, it is azimuthal, but when "quadratic" or "tangent" is chosen, it is not azimuthal.   Should it be classified as miscellaneous?

Reply:
> Should it be classified as miscellaneous?

yes, you could add the precision about when it is azimuthal or not later in the file